test/uk/gov/hmrc/helptosave/utils/MongoSupport.scala

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
you can move this to `repo` package

Reply:
I think utils is more suited?

Reply:
its being used by only mongo specs, so no point in keeping in util package

Reply:
ok I've moved it.